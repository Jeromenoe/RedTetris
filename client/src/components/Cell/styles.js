import styled from 'styled-components'

export const StyledCell = styled.div`
	background: #${props => props.color}
`