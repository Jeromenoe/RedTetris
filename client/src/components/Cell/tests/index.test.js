import React from 'react';
import Cell from '../index';
import renderer from 'react-test-renderer';


describe('Testing Cell Component', () => {
	test('Check valids parameters', async () => {
		const tree = renderer.create(<Cell type={"T"} color={"#000000"}/>).toJSON();
		expect(tree).toMatchSnapshot();
	  });
	test('Check with spectrum type', async () => {
		const tree = renderer.create(<Cell type={"spectrum"} color={"#000000"}/>).toJSON();
		expect(tree).toMatchSnapshot();
	  });
});