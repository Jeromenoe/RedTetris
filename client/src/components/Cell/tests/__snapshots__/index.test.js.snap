// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Cell Component Check valids parameters 1`] = `
<div
  className="sc-bdnxRM dCXbye"
  color="76448a"
  type="T"
/>
`;

exports[`Testing Cell Component Check with spectrum type 1`] = `
<div
  className="sc-bdnxRM dIXOCh"
  color="148f77"
  type="spectrum"
/>
`;
